mcs/class/System.Net.Http/Test/System.Net.Http/HttpClientTest.cs

```diff
@@ -1459,6 +1456,7 @@ public void WildcardConnect ()
 				Assert.That (wex.Status, Is.EqualTo (WebExceptionStatus.ConnectFailure), "#3");
```

Comment:
The proper fix would've been to check that `e.InnerException` is PlatformNotSupportedException.

Reply:
@akoeplinger please submit a PR